### PR TITLE
update api references

### LINKED ISSUE #1857: Updates to API references
- [x] [Version object](https://docs.meilisearch.com/reference/api/version.html#version-object) -> Bold "Name" column 
- [x] [Typo tolerance object](https://docs.meilisearch.com/reference/api/settings.html#typo-tolerance-object) -> Use `Empty` instead of `[]`
- [x] Update the router element colors. [Patch](https://docs.meilisearch.com/reference/api/keys.html#update-a-key) and [get](https://docs.meilisearch.com/reference/api/keys.html#get-all-keys) have the same green

### PR DESCRIPTION
**Pull request by maryamsulemani97**
closes #1857 + #1948

~~If anyone has any other color suggestions, we can change this~~
Updated to [Swagger's](https://github.com/swagger-api/swagger-ui/blob/6ae2693d47397a919370eb19149f5514cb2f99f2/src/style/_variables.scss#L51) colors

### CODE REVIEW

File: .vuepress/components/RouteHighlighter.vue [hunk 1]
```diff
@@ -72,6 +75,11 @@ div.route {
   color: white;
 }
 
+.route div.method.patch {
+  background-color: #968efa;
```
Comment (maryamsulemani97) [inline .vuepress/components/RouteHighlighter.vue:79]:
```suggestion
  background-color: #d93988;
```

**Comment by guimachiavelli** [comment]
What about we follow [swagger's colour coding](https://github.com/swagger-api/swagger-ui/blob/6ae2693d47397a919370eb19149f5514cb2f99f2/src/style/_variables.scss#L51) on this?

**Review by guimachiavelli — APPROVED** [review]
💺

**Comment by maryamsulemani97** [comment]
bors merge

**Pull request merged**